src/router.tsx

```diff
@@ -5,10 +5,10 @@ import PlayPageRouter from "./pages/Play/router"
 const MainRouter: React.FC = () => {
   return (
     <>
-    <MainPageRouter />
-    <Switch>
-      <PlayPageRouter />
-    </Switch>
+      <Switch>
+        <MainPageRouter />
+        <PlayPageRouter />
+      </Switch>
```

Comment:
```tsx
const MainPageRouter: React.FC = () => {
  return <Route exact path="/" component={Main}/>
}
```

> https://github.com/riemannulus/umamusume-client/blob/4317b9f/src/pages/Main/router.tsx#L5

`exec` prop 가 있어 괜찮지 않나요? 아니면 루트 경로를 마지막에 적어두면 괜찮을까요?

```suggestion
      <Switch>
        <PlayPageRouter />
        <MainPageRouter />
      </Switch>
```

Reply:
한번 로컬에서 고쳐서 해보시겠어요?

Reply:
순서를 바꾸는 경우 `Main` 컴포넌트를 불러우지 않습니다. `exact` props 를 빼보았으나 결과는 같습니다.

지금 버전 4317b9f 은 `exact` pros 덕에 라우팅이 정상적으로 됩니다.

Reply:
https://deploy-preview-7--dreamy-northcutt-cfb2f0.netlify.app/ 지금 버전 프리뷰에서는 정작 play 페이지가 제대로 라우팅이 안 되는 것 같네요 🤔